eng/pipelines/build.yml

```diff
@@ -133,6 +133,9 @@ jobs:
       - _TestArgs: '-test'
       - _Cross: ''
 
+      - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
+        - _buildLog: '-binaryLog'
```

Comment:
Let's just inline this to the call after CI unconditionally. It's arguably a bug in arcade (win always produces unless we exclude, and linux only if you include)